app/Controllers/Items.php

```
@@ -1099,87 +1096,98 @@ private function validateCSVStockLocations(array $row, array $allowedLocations):
-        foreach ($check_for_numeric_values as $key => $value) {
+        foreach ($valuesToCheckForNumeric as $key => $value) {
             if (!is_numeric($value) && !empty($value)) {
                 log_message('error', "non-numeric: '$value' for '$key' when numeric is required");
                 return true;
             }
         }
 
+        // Check stock locations
+        $invalidLocations = $this->validateCSVStockLocations($row, $allowedStockLocations);
+        if (!empty($invalidLocations)) {
+            $isFailedRow = true;
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$isFailedRow'. (undefined)

(UnusedLocalVariable)